docs/01-app/01-getting-started/05-css-and-styling.mdx

```diff
@@ -0,0 +1,542 @@
+---
+
+Next.js provides several ways to use CSS in your application, including:
+
+- [CSS Modules](#css-modules)
+- [Global CSS](#global-css)
+- [Tailwind CSS](#tailwind-css)
+- [Sass](#sass)
+- [CSS-in-JS](#css-in-js)
+- [External Stylesheets](#external-stylesheets)
+
+This page will guide you through how to use each of these approaches.
+
+## CSS Modules
+
+CSS Modules locally scope CSS by generating unique class names. This allows you to use the same class in different files without worrying about collisions.
+
+To start using CSS Modules, create a new file with the extension `.module.css` and import it into any component inside the `app` directory:
+
+```css filename="app/blog/styles.module.css"
+.blog {
+  padding: 24px;
+}
+```
+
+```tsx filename="app/blog/page.tsx" switcher
+import styles from './styles.module.css'
+
+export default function Page({ children }: { children: React.ReactNode }) {
+  return <main className={styles.blog}>{children}</main>
+}
+```
+
+```jsx filename="app/blog/page.js" switcher
+import styles from './styles.module.css'
+
+export default function Page({ children }) {
+  return <main className={styles.blog}>{children}</main>
+}
+```
+
+> **Good to know:**
+>
+> - CSS Modules are only enabled for files with the `.module.css` and `.module.sass` extensions.
+> - In production, all CSS Module files will be automatically concatenated into minified and code-split `.css` files.
+> - These `.css` files represent hot execution paths in your application, ensuring the minimal amount of CSS is loaded for your application to paint.
+
+## Global CSS
+
+You can use global CSS to apply styles across your application.
+
+To use global styles, create a new CSS file, for example `app/global.css`:
+
+```css filename="app/global.css"
+body {
+  padding: 20px 20px 60px;
+  max-width: 680px;
+  margin: 0 auto;
+}
+```
+
+Import the file in the root layout (`app/layout.js`) to apply the styles to **every route** in your application:
+
+```tsx filename="app/layout.tsx" switcher
+// These styles apply to every route in the application
+import './global.css'
+
+export default function RootLayout({
+  children,
+}: {
+  children: React.ReactNode
+}) {
+  return (
+    <html lang="en">
+      <body>{children}</body>
+    </html>
+  )
+}
+```
+
+```jsx filename="app/layout.js" switcher
+// These styles apply to every route in the application
+import './global.css'
+
+export default function RootLayout({ children }) {
+  return (
+    <html lang="en">
+      <body>{children}</body>
+    </html>
+  )
+}
+```
+
+> **Good to know:** Global styles can be imported into any layout, page, or component inside the `app` directory. However, since Next.js uses React's built-in support for stylesheets to integrate with Suspense. This built-in support currently does not remove stylesheets as you navigate between routes. Therefore, we recommend using global styles for _truly_ global CSS, and C[SS Modules](#css-modules) for scoped CSS.
```

Comment:
```suggestion
> **Good to know:** Global styles can be imported into any layout, page, or component inside the `app` directory. However, since Next.js uses React's built-in support for stylesheets to integrate with Suspense. This built-in support currently does not remove stylesheets as you navigate between routes. Therefore, we recommend using global styles for _truly_ global CSS, and [CSS Modules](#css-modules) for scoped CSS.
```